x-pack/plugins/secops/public/components/drag_and_drop/draggable_wrapper.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { defaultTo } from 'lodash/fp';
+import * as React from 'react';
+import { Draggable, Droppable } from 'react-beautiful-dnd';
+import { connect } from 'react-redux';
+import styled from 'styled-components';
+import { ActionCreator } from 'typescript-fsa';
+
+import { dragAndDropActions } from '../../store/local/drag_and_drop';
+import { IdToDataProvider } from '../../store/local/drag_and_drop/model';
+import { dataProvidersSelector } from '../../store/local/drag_and_drop/selectors';
+import { State } from '../../store/reducer';
+import { DataProvider, DataProviderId } from '../timeline/data_providers/data_provider'; // tslint:disable-line:no-unused-variable (linter bug!)
```

Comment:
How about we go ahead and just remove:
```
  no-unused-variable: true
```

from: tslint.yaml

And let the server builds fail if we forget something since that has been deprecated. It served us well, but it's not worth adding these lines now.